src/crypto_backends/openssl.rs

```diff
@@ -100,6 +87,28 @@ impl LocalKeyPair for OpenSSLLocalKeyPair {
             pub_key_point.to_bytes(&GROUP_P256, PointConversionForm::UNCOMPRESSED, &mut bn_ctx)?;
         Ok(uncompressed)
     }
+
+    fn from_raw_components(components: &EcKeyComponents) -> Result<Self> {
+        let d = BigNum::from_slice(&components.d)?;
+        let mut bn_ctx = BigNumContext::new()?;
+        let ec_point = EcPoint::from_bytes(&GROUP_P256, &components.public_key, &mut bn_ctx)?;
```

Comment:
Based on curve discussions in https://github.com/mozilla-services/a-s-private/pull/6/files/3c6d0a092f461795b3c50ed455209fc54fd2f223#diff-ce9ee613758355e3cb09712f34f737e8R251, does it make sense to add a `Default` for `EcKeyComponents` and specify the curve there?